Source/WebCore/Modules/beacon/NavigatorBeacon.cpp

```diff
@@ -67,7 +67,7 @@ ASCIILiteral NavigatorBeacon::supplementName()
     return "NavigatorBeacon"_s;
 }
 
-void NavigatorBeacon::notifyFinished(CachedResource& resource, const NetworkLoadMetrics&)
+void NavigatorBeacon::notifyFinished(CachedResource& resource, const NetworkLoadMetrics&, bool)
```

Comment:
I think we prefer using `enum class` for clarity

Reply:
I'll use LoadWillContinueInAnotherProcess